src/custom/api/gnosisProtocol/api.ts

```diff
@@ -481,7 +480,25 @@ export interface GasFeeEndpointResponse {
   fastest: string
 }
 
+// Hardcoding gas prices for XDAI/GCHAIN
+// 1 gwei the lowest to 20 gwei the highest
+const HARDCODED_XDAI_GAS = {
+  lastUpdate: '',
+  lowest: '1000000000',
+  safeLow: '2000000000',
+  standard: '4000000000',
+  fast: '4000000000',
+  fastest: '2000000000',
+}
+
+function _getGChainGasPrices() {
+  return { ...HARDCODED_XDAI_GAS, lastUpdate: new Date().toISOString() }
+}
+
 export async function getGasPrices(chainId: ChainId = DEFAULT_NETWORK_FOR_LISTS): Promise<GasFeeEndpointResponse> {
+  if (chainId === SupportedChainId.XDAI) {
+    return _getGChainGasPrices()
```

Comment:
should we try first to ask the node and/or a relevant service?
* https://docs.ethers.io/v5/api/providers/provider/#Provider-getGasPrice
* https://www.xdaichain.com/for-developers/developer-resources/gas-price-oracle

Reply:
Will give a try to ethers gas estimation.
And also replace how mainnet/rinkeby calculates gas, but to reduce the impact only for claim.
We can revisit later if we want to do that also for the swap